### PR TITLE
slownik pojec

### PR DESCRIPTION
**Pull request by pierewoj**

### CODE REVIEW

File: docs/licencjat.tex [hunk 1]
```diff
@@ -246,6 +246,21 @@ \chapter{Conclusions}
 \chapter{Team members contribution}
 	\krystynaintodo{Podział prac}
 
+\chapter{Dictionary}
```
Comment (gajczix) [inline docs/licencjat.tex]: Brakuje tam kilku pojęć takich jak: aktywacja, batch normalizacja, Relu, epoch, dropout (Adam używa), backprop, forward prop. Słownik powinien być na początku, zaraz po rozdziale ze wstępem.

File: docs/licencjat.tex [hunk 2]
```diff
@@ -246,6 +246,21 @@ \chapter{Conclusions}
...
+    \begin{itemize}
+        \item Neural network - a machine learning model, inspired by a structure of a human brain. It consits of a large collection of neurons grouped into layers. Each neuron is connected to a set of other neurons. Each connection has an assigned value (also known as weight), which measures its importance.
+
+        \item Training - a process which consists of modification of connection weights after processing input data by a neural network and comparing its output with the desired output.
+
+        \item Inference - a process of computing neural network output based on the input and values of neuron weights
+
+        \item Convolution (discrete) - a mathematical function defined as follows
```
Comment (gajczix) [inline docs/licencjat.tex]: Napisz najpierw o zwykłej konwolucji, a potem o tej dyskretnej. Dobry pomysł z umieszczeniem tego w słowniczku, można usunąć TODO z tłumaczeniem co to jest dyskretna konwolucja.

File: docs/licencjat.tex [hunk 3]
```diff
@@ -246,6 +246,21 @@ \chapter{Conclusions}
 \chapter{Team members contribution}
 	\krystynaintodo{Podział prac}
 
+\chapter{Dictionary}
+    \begin{itemize}
```
Comment (gajczix) [inline docs/licencjat.tex]: Przed tym krótkie zdanie "dla wygody czytelnika podajemy przydatne terminy" czy coś w tym stylu

File: docs/licencjat.tex [hunk 4]
```diff
@@ -246,6 +246,21 @@ \chapter{Conclusions}
...
+
+        \item Training - a process which consists of modification of connection weights after processing input data by a neural network and comparing its output with the desired output.
+
+        \item Inference - a process of computing neural network output based on the input and values of neuron weights
+
+        \item Convolution (discrete) - a mathematical function defined as follows
+        $$ (f\star g)(n) = \sum_{m=-\infty}^{\infty} f[m]*g[n-m]$$ where $$f,g\in Z\rightarrow Z$$
+
+        \item Convolutonal layer - a neural network layer which performs a convolution operation between its inputs and weights (in this case they are usually called weight filters or kernels). Usage of this kind of layers proved to be successful in the field of image recognition, video recognition and natural language processing.
+		\krystynaintodo{dodać więcej}
```
Comment (gajczix) [inline docs/licencjat.tex]: wymieniłam te na pewno potrzebne, jak coś się nowego wymyśli to dodamy jak już praca będzie gotowa

**Review by gajczix — CHANGES_REQUESTED** [review]

File: docs/licencjat.tex [hunk 5]
```diff
@@ -57,46 +57,80 @@ \section{What is deep learning and why is it interesting?}
...
+        \item Mini batch - a small subset of input data. It is usually a good practice to perform weight update after processing a mini batch instead of a single element of the training set.
+        
+        \item Batch normalization layer - a layer which returns a modification of its input such 
+        that it has zero mean and unit variance.
+        
+        \item ReLU - a neuron activation function defined as follows: $$ R(x) = max(0,x) $$
+        
+        \item Epoch - one pass over all training set.
+        
+        \item Dropout layer - a layer which returns a modification of its input such that some of the values are randomly set to zero. Dropout layers are used as a regularization technique.
```
Comment (gajczix) [inline docs/licencjat.tex]: explain regularization

**Review by gajczix — APPROVED** [review]
